doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -128,14 +128,54 @@ Monitor the log file in a new terminal with:
 
     tail -f convert-gaia.log
 
-Check the log ouput after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
+Check the log output after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
 
 .. _lsst.meas.algorithms-refcat-ingest:
 
 4. Ingest the files into the butler
 ===================================
 
-When ``convertReferenceCatalog`` has finished, it will print the two commands you need to run to register the new refcat dataset type and ingest your converted output into it.
+When ``convertReferenceCatalog`` has finished, a new directory (named ``gaia-refcat/`` in the example above) will now exist containing all reference catalogs in the LSST format.
+
+.. note::
+
+    If you have an already existing converted reference catalog on disk (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers), you will first need to create a suitable filename to htm7-index astropy lookup table before running the commands in the remainder of this section.
+
+    This is an example script that constructs conversion lookup tables for all converted FITS files in the `gaiadr2` and `panstarrsps1` directories, outputting to the `outdir` directory:
+
+    .. code-block:: python
+
+        import os
+        import glob
+        import astropy.table
+
+        refcat_dirs = [gaiadr2, panstarrsps1]
+
+        # loop over each FITS file in all refcats
+        # note: this constructs a series of .ecsv files, each containing two columns:
+        # 1) the FITS filename, and 2) the htm7 pixel index
+        for refcat_dir in refcat_dirs:
+
+            outfile = f"{outdir}/{os.path.basename(refcat_dir)}.ecsv"
+            print(f"Saving to: {outfile}")
+
+            table = astropy.table.Table(names=("filename", "htm7"), dtype=("str", "int"))
+            files = glob.glob(f"{refcat_dir}/[0-9]*.fits")
+
+            for ii, file in enumerate(files):
+                print(f"{ii}/{len(files)} ({100*ii/len(files):0.1f}%)", end="\r")
```

Comment:
I don't think you want a print here at all: it would print ~65k lines. I don't think it's necessary; how long does it take to run, anyway?